bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.csproj

```diff
@@ -64,6 +64,10 @@
       <Pack>true</Pack>
       <PackagePath>runtimes/win/native</PackagePath>
     </Content>
+    <Content Include="$(LibMongoCryptBinaries)/libmongocrypt.so">
```

Comment:
Why is it called `mongocrypt.dll` on Windows (line 63) but called `libmongocrypt.so` on Linux?

Reply:
This is how it's named via c team. As @kevinAlbs  told me, most likely it's related to a Windows convention, not to prefix dlls with “lib”. There was a ticket for this: https://jira.mongodb.org/browse/CDRIVER-3449

Reply:
OK... very confusing.

Reply:
Linux uses a naming convention of `lib<<name>>.so` whereas Windows uses `<<name>>.dll`. The Linux Loader provides some additional features beyond simply locating the shared library. If you're really curious the following is a good read:

https://tldp.org/HOWTO/Program-Library-HOWTO/shared-libraries.html

Reply:
And if/when we support OSX, we will have to deal with loading the `dylib` - which is what OSX calls DLLs and SOs.

Reply:
as for `OSX`, this is in the scope of my current ticket https://jira.mongodb.org/browse/MONGOCRYPT-174